src/win/win-helpers.cpp

```diff
@@ -42,6 +42,8 @@ namespace librealsense
 {
     namespace platform
     {
+        template<typename T> size_t vector_bytes_size(const typename std::vector<T>& vec) { return sizeof(T) * vec.size(); }
```

Comment:
To be on the save side pls limit/specialize it to built-in types only to avoid anomalies